en/guide/xml_schema.md

```diff
@@ -133,18 +133,26 @@ MAV_CMD entry `value` elements may additionally define these tags/fields:
 ### param {#param}
 
 The `<param>` tag is used in the [MAV_CMD](../messages/common.md#MAV_CMD) enum as part of defining mission commands.
-Params must include an attribute `index` (with value from 1 to 7) and contain a parameter description string.
+Each entry value may have up to 7 params declared, with `index` values from 1-7. 
 
-A `param` may also include the following optional attributes (which may be used by a GUI for parameter display and editing):
+A `param` **must** include the following attribute:
+- `index` - The parameter number (1 - 7).
+
+A `param` **should** have:
+- `description`: Parameter description string (tag body)
+
+A `param` **should** also include the following optional attributes where appropriate (which may be used by a GUI for parameter display and editing):
 - `label` - Display name to represent the parameter in a GCS or other UI
 - `units` - SI units for the value.
 - `enum` - Enum containing possible values for the parameter (if applicable).
 - `decimalPlaces` - Hint to a UI about how many decimal places to use if the parameter value is displayed.
 - `increment` - Allowed increments for the parameter value.
 - `minValue` - Minimum value for param.
 - `maxValue` - Maximum value for the param.
-
-
+- `reserved` - Param is reserved for future use. Default is `False`.
```

Comment:
I had to stare at this for a bit to stop being confused. Specifically the ```Default is `False`.``` part. I was thinking that was talking about the default value for the param itself. Not sure if that's just me.

Reply:
If you can find it confusing, then someone else will too. Reworded:

> Boolean indicating whether param is reserved for future use. If the attributes is not declared, then implicitly `reserved="False".